diff --git a/env-example b/env-example
index 0dbef66..eff8dc9 100644
--- a/env-example
+++ b/env-example
@@ -50,7 +50,7 @@ PHP_INTERPRETER=php-fpm
 ### Docker Host IP ########################################
 
 # Enter your Docker Host IP (will be appended to /etc/hosts). Default is `10.0.75.1`
-DOCKER_HOST_IP=10.0.75.1
+DOCKER_HOST_IP=192.168.3.43
 
 ### Remote Interpreter ####################################
 
@@ -82,18 +82,18 @@ WORKSPACE_COMPOSER_GLOBAL_INSTALL=true
 WORKSPACE_COMPOSER_REPO_PACKAGIST=https://packagist.laravel-china.org
 WORKSPACE_INSTALL_NODE=true
 WORKSPACE_NODE_VERSION=node
-WORKSPACE_NPM_REGISTRY=
+WORKSPACE_NPM_REGISTRY=https://registry.npm.taobao.org
 WORKSPACE_INSTALL_YARN=true
 WORKSPACE_YARN_VERSION=latest
 WORKSPACE_INSTALL_NPM_GULP=true
 WORKSPACE_INSTALL_NPM_BOWER=false
 WORKSPACE_INSTALL_NPM_VUE_CLI=true
 WORKSPACE_INSTALL_PHPREDIS=true
-WORKSPACE_INSTALL_WORKSPACE_SSH=false
+WORKSPACE_INSTALL_WORKSPACE_SSH=true
 WORKSPACE_INSTALL_SUBVERSION=false
-WORKSPACE_INSTALL_XDEBUG=false
+WORKSPACE_INSTALL_XDEBUG=true
 WORKSPACE_INSTALL_PHPDBG=false
-WORKSPACE_INSTALL_SSH2=false
+WORKSPACE_INSTALL_SSH2=true
 WORKSPACE_INSTALL_LDAP=false
 WORKSPACE_INSTALL_GMP=false
 WORKSPACE_INSTALL_SOAP=false
@@ -136,18 +136,18 @@ PHP_FPM_INSTALL_BCMATH=true
 PHP_FPM_INSTALL_MYSQLI=true
 PHP_FPM_INSTALL_TOKENIZER=true
 PHP_FPM_INSTALL_INTL=true
-PHP_FPM_INSTALL_IMAGEMAGICK=true
+PHP_FPM_INSTALL_IMAGEMAGICK=false
 PHP_FPM_INSTALL_OPCACHE=true
 PHP_FPM_INSTALL_IMAGE_OPTIMIZERS=true
 PHP_FPM_INSTALL_PHPREDIS=true
 PHP_FPM_INSTALL_MEMCACHED=false
-PHP_FPM_INSTALL_XDEBUG=false
+PHP_FPM_INSTALL_XDEBUG=true
 PHP_FPM_INSTALL_PHPDBG=false
 PHP_FPM_INSTALL_IMAP=false
 PHP_FPM_INSTALL_MONGO=false
 PHP_FPM_INSTALL_AMQP=false
 PHP_FPM_INSTALL_MSSQL=false
-PHP_FPM_INSTALL_SSH2=false
+PHP_FPM_INSTALL_SSH2=true
 PHP_FPM_INSTALL_SOAP=false
 PHP_FPM_INSTALL_GMP=false
 PHP_FPM_INSTALL_EXIF=false
@@ -171,7 +171,7 @@ PHP_WORKER_INSTALL_BCMATH=false
 
 ### NGINX #################################################
 
-NGINX_HOST_HTTP_PORT=80
+NGINX_HOST_HTTP_PORT=88
 NGINX_HOST_HTTPS_PORT=443
 NGINX_HOST_LOG_PATH=./logs/nginx/
 NGINX_SITES_PATH=./nginx/sites/
diff --git a/php-fpm/xdebug.ini b/php-fpm/xdebug.ini
index c3f32ec..a9bdafd 100644
--- a/php-fpm/xdebug.ini
+++ b/php-fpm/xdebug.ini
@@ -1,14 +1,14 @@
 ; NOTE: The actual debug.so extention is NOT SET HERE but rather (/usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini)
 
-; xdebug.remote_host=dockerhost
-xdebug.remote_connect_back=1
+xdebug.remote_host=dockerhost
+xdebug.remote_connect_back=0
 xdebug.remote_port=9000
 xdebug.idekey=PHPSTORM
 
 xdebug.remote_autostart=0
 xdebug.remote_enable=0
 xdebug.cli_color=0
-xdebug.profiler_enable=0
+xdebug.profiler_enable=1
 xdebug.profiler_output_dir="~/xdebug/phpstorm/tmp/profiling"
 
 xdebug.remote_handler=dbgp
@@ -18,3 +18,4 @@ xdebug.var_display_max_children=-1
 xdebug.var_display_max_data=-1
 xdebug.var_display_max_depth=-1
 
+xdebug.remote_log ="/tmp/xdebug.log"
\ No newline at end of file
diff --git a/workspace/xdebug.ini b/workspace/xdebug.ini
index c3f32ec..07b0149 100644
--- a/workspace/xdebug.ini
+++ b/workspace/xdebug.ini
@@ -1,14 +1,14 @@
 ; NOTE: The actual debug.so extention is NOT SET HERE but rather (/usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini)
 
-; xdebug.remote_host=dockerhost
-xdebug.remote_connect_back=1
+xdebug.remote_host=dockerhost
+xdebug.remote_connect_back=0
 xdebug.remote_port=9000
 xdebug.idekey=PHPSTORM
 
 xdebug.remote_autostart=0
 xdebug.remote_enable=0
 xdebug.cli_color=0
-xdebug.profiler_enable=0
+xdebug.profiler_enable=1
 xdebug.profiler_output_dir="~/xdebug/phpstorm/tmp/profiling"
 
 xdebug.remote_handler=dbgp
@@ -18,3 +18,4 @@ xdebug.var_display_max_children=-1
 xdebug.var_display_max_data=-1
 xdebug.var_display_max_depth=-1
 
+xdebug.remote_log ="/tmp/xdebug.log"
